<launch>
    <rosparam file="$(find luxo_hardware_interface)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find luxo_hardware_interface)/config/controllers.yaml" command="load"/>
    <rosparam file="$(find luxo_hardware_interface)/config/joint_limits.yaml" command="load"/>
    <param name="robot_description" textfile="$(find luxo_hardware_interface)/config/luxo.urdf"/>
    <node name="luxo_hardware_interface" pkg="luxo_hardware_interface" type="luxo_hardware_interface_node" output="screen" respawn="true" respawn_delay="2"/>

    <node name="controller_spawner" pkg="controller_manager" type="spawner" output="screen" ns="/"
        args="
            /luxo/controller/state
            /luxo/controller/trajectory
        " respawn="true" respawn_delay="2"/>

    <!-- 
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/"
        args="
            /luxo/controller/state
            /luxo/controller/position/foot
            /luxo/controller/position/knee
            /luxo/controller/position/ass
            /luxo/controller/position/neck
            /luxo/controller/position/head
        "/>
    -->

</launch>
